cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -395,6 +391,9 @@ instance ( ShelleyBasedEra era
   toObject _verb InvalidMetadata =
     mkObject [ "kind" .= String "InvalidMetadata"
              ]
+  toObject _verb (ExtraneousScriptWitnessesUTXOW _) =
+    mkObject [ "kind" .= String "ExtraneousScriptWitnessesUTXOW"
```

Comment:
This drops the extra information provided - we should also print the set of extra scripts.